packages/lavapack/package.json

```diff
@@ -32,7 +33,7 @@
     "lint:eslint": "eslint .",
     "lint:fix": "eslint . --fix",
     "lint:deps": "depcheck",
-    "test": "ava --timeout=30s test/index.js",
+    "test": "ava --timeout=30s test/index.js && node test/runtimeEcma.js",
```

Comment:
I recommend exporting to a separate yarn command and then have another main one that combines the two

Reply:
ok